src/main/java/org/beanpod/switchboard/dto/mapper/DateMapper.java

```diff
@@ -0,0 +1,21 @@
+package org.beanpod.switchboard.dto.mapper;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Date;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface DateMapper {
```

Comment:
Why didn't we need this before?

Reply:
Because the previously when we had only DTOs and Entities they both used `java.util.Date` for their last communications, but Open api spec uses `java.time.OffsetDatetime` for timestamps.